src/rp2_common/pico_stdio_usb/include/pico/stdio_usb.h

```diff
@@ -42,6 +44,36 @@
 #define PICO_STDIO_USB_LOW_PRIORITY_IRQ 31
 #endif
 
+// PICO_CONFIG: PICO_STDIO_USB_ENABLE_RESET_VIA_BAUD_RATE, Enable/disable resetting into BOOTSEL mode if the host sets the baud rate to a magic value (PICO_STDIO_USB_RESET_MAGIC_BAUD_RATE), type=bool, default=1, group=pico_stdio_usb
+#ifndef PICO_STDIO_USB_ENABLE_RESET_VIA_BAUD_RATE
+#define PICO_STDIO_USB_ENABLE_RESET_VIA_BAUD_RATE 1
```

Comment:
Does it definitely make sense to enable-this-by-default, given your comments in #112 about "the OS is understandably not keen on you trying to talk CDC commands directly to a serial device it has claimed"? :man_shrugging:

Reply:
that referred to trying to talk to a CDC device via libusb while the OS has it claime; you can use OS level things to set the baud rate just fne